git-open

```diff
@@ -235,21 +235,16 @@ case $( uname -s ) in
   CYGWIN*)  open='cygstart';;
   *)        # Try to detect WSL (Windows Subsystem for Linux)
             if uname -r | grep -q Microsoft; then
-              open='powershell.exe'
-              openopt='Start'
+              open='powershell.exe Start'
             else
               open='xdg-open'
             fi;;
 esac
 
 # Allow printing the url if BROWSER=echo
-if [[ $BROWSER == "echo" ]]; then
-  openopt=''
-else
+if [[ $BROWSER != "echo" ]]; then
   exec &>/dev/null
 fi
 
 # open it in a browser
-${BROWSER:-$open} $openopt "$openurl"
-
-unset openopt
+eval "${BROWSER:-$open} $openurl"
```

Comment:
You should be able to just have:


```shell
${BROWSER:-$open} $openurl
```

Reply:
The `eval` is required to make commands with a parameter (such as `powershell.exe Start`) execute properly on my system. Is it not that way for you?

Reply:
yeah works fine for me. 
![desktop](https://user-images.githubusercontent.com/999489/48867190-62552700-eda3-11e8-8222-a3eed3dd284c.png)

What version of bash are you using?

Reply:
You're right... I had some sort of weird bug that went away when I clobbered the file, without making any changes. I have no idea what happened, but I've updated the PR.